src/main/java/com/example/demo/util/BackupGeometryUtil.java

```diff
@@ -0,0 +1,47 @@
+package com.example.demo.util;
+
+import com.example.demo.service.AmazonS3Service;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+@Component
+@EnableScheduling
+public class BackupGeometryUtil {
+    private static final String FILE_NAME = "mytab.sql";
+    private final AmazonS3Service amazonS3Service;
+    private final String password;
+
+    public BackupGeometryUtil(AmazonS3Service amazonS3Service,
+                              @Value("${spring.datasource.password}") String password) {
+        this.amazonS3Service = amazonS3Service;
+        this.password = password;
+    }
+
+    @Scheduled(cron = "* */15 * * * *")
+    public void backupGeometries() throws IOException, InterruptedException {
```

Comment:
Can be in the service where you working with GeoJson geometries. Or rename this one